src/OpenTelemetry.Contrib.Instrumentation.Elasticsearch/Implementation/ElasticsearchActivitySourceHelper.cs

```diff
@@ -0,0 +1,36 @@
+﻿// <copyright file="ElasticsearchActivitySourceHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Diagnostics;
+using System.Reflection;
+
+namespace OpenTelemetry.Contrib.Instrumentation.ElasticsearchClient.Implementation
+{
+    /// <summary>
+    /// Helper class to hold common properties used by both ElasticsearchRequestPipelineDiagnosticListener.
+    /// </summary>
+    internal class ElasticsearchActivitySourceHelper
```

Comment:
nit: this class doesn't contain any helper method, so the old name "constants" was still good.

Reply:
I'm going to remove this class and add these constants to the `ElasticsearchRequestPipelineDiagnosticListener` class.